### PR TITLE
gateio: (minor) do not crash in parseOrder() on empty timestamp 

### PR DESCRIPTION
**Pull request by egorFiNE**

### CODE REVIEW

**Comment by egorFiNE** [comment]
Could you perhaps create label "minor" for changes that needs very simple and quick code review? Sort of like "minor edit" in wikipedia or something?..

File: js/gateio.js [hunk 1]
```diff
@@ -378,7 +381,7 @@ module.exports = class gateio extends Exchange {
         }
         return {
             'id': id,
-            'datetime': this.iso8601 (timestamp),
```
Comment (kroitor) [inline js/gateio.js]: I'm trying to understand why it did crash there, whereas it actually should've worked ok... the `iso8601` method is designed to work with undefined values, unless there was some other type of value in the `timestamp` (not undefined, but `null`, maybe?)

- https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L185
- https://github.com/ccxt/ccxt/blob/master/python/ccxt/base/exchange.py#L669
- https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L489

**Comment by egorFiNE** [comment]
It crashes in multiple exchanges and multiple cases. 

I thought as well to update the base method. Shall we?

**Comment by egorFiNE** [comment]
And `null`, yes. It does happen.

**Comment by kroitor** [comment]
> I thought as well to update the base method. Shall we?

That would be awesome!

**Comment by egorFiNE** [comment]
> That would be awesome!

Here you go: #3043. 

PHP and Python versions will follow today from my coworkers with similar branch names.

**Pull request merged**